packages/block-library/src/group/index.js

```diff
@@ -22,53 +22,49 @@ export { metadata, name };
 export const settings = {
 	icon,
 	example: {
-		innerBlocks: [
-			{
-				name: 'core/paragraph',
-				attributes: {
-					customTextColor: '#cf2e2e',
-					fontSize: 'large',
-					content: __( 'One.' ),
-				},
+		attributes: {
+			layout: {
+				type: 'constrained',
```

Comment:
Simple and looking pretty good. 

https://github.com/WordPress/gutenberg/assets/6458278/b2bb9810-6e36-407c-a734-764201088ad0